documents/ITM-0100.txt

```diff
@@ -112,9 +112,11 @@ Status: draft
 
 2.第三阶段：2021-至今
 
-2021年，UI和APP最后留下的人并不多，都是独苗子，只有游戏和程序部稍微人多点。同时不幸的是，爱特被纳入了学部创想会旗下，成为了“校级”社团（徒有其表）；另外在2022年，爱特随着信部搬来了荒凉的西海岸，场地一直没着落，直到最近五八工坊建成才有了工位。但是从大趋势看，近几年爱特成员质量出现下滑趋势，这是第一个头大的问题，第二个问题是团委太功利了，不是很待见没有量化成果（获奖）的爱特，各种支持给的很少，于是我们选择拥抱学院，向曲老师寻求更好的指导，并且在他的建议下，在2023年成立了FOSS部门，致力于发展海大的开源事业。
-2024年暑假，由于创想会内部调整，决定减少旗下社团数量，最终原 iOS Club 因正式社员人数过少(当时仅余2人)，并入爱特工作室作为第六部门 iOS部。 
-现在是大模型的时代，可能以后还会开展人工智能相关的部门，不过，这些都是后话了。要相信后人的智慧。
+-2021年，UI和APP最后留下的人并不多，都是独苗子，只有游戏和程序部稍微人多点。同时不幸的是，爱特被纳入了学部创想会旗下，成为了“校级”社团（徒有其表）；
+-2022年，爱特随着信部搬来了荒凉的西海岸，场地一直没着落，直到9月五八工坊建成才有了工位。但是从大趋势看，近几年爱特成员质量出现下滑趋势，这是第一个头大的问题，根本的原因是新生和老生在不同的校区，这给社团的管理带来了很大的问题，校区之间相隔太远，不可能频繁的让老生往返于西海岸和崂山，新生和老生的交流大多在线上，新生之间的见面也只有例会，这样新生之间的联系也没有之前这么紧密，2023和2024年在工位常驻的小登们也只有站长，不得不让人担心。第二个问题是团委太“功利”了，不是很待见没有量化成果（获奖）的爱特，各种支持给的很少，于是我们选择拥抱学院，向曲老师寻求更好的指导，并且在他的建议下，在2023年成立了FOSS部门，致力于发展海大的开源事业。
```

Comment:
From the point of view of history writing, isn't it a bit strange to have the years 2023 and 2024 in the records of 2022?

Reply:
Will there be projects in the FOSS department? 🥲

Reply:
Fixed, I do not know the details of FOSS. Maybe it will.